src/rowdy/RowdyRunner.java

```diff
@@ -266,11 +267,6 @@ public void executeStmt(Node parent, Node seqControl) {
           }
           break;
         case FUNC_CALL:
-          String funcName = ((Terminal) currentTreeNode.get(ID).symbol()).getName();
```

Comment:
This was removed because if a user, while in the command line, it's possible a main is defined. It's ok to call main